src/ducks/Watchlists/Actions/Edit/EditForm/Assets.js

```diff
@@ -0,0 +1,185 @@
+        item.ticker.toLowerCase().includes(_filter)
+      _items = _items.filter(filterHelper)
+    }
+    setItems(_items)
+  }, [filter, watchListItems])
+
+  const checkboxClickHandler = (item, newValue) =>
+    setCheckedItems(old => {
+      const items = [...old]
+      const index = items.findIndex(i => i.id === item.id)
+      if (index === -1 && newValue) {
+        items.push(item)
+      } else if (index > -1 && !newValue) {
+        items.splice(index, 1)
+      }
+      if (onChange) onChange(items)
+      return items
+    })
+
+  return (
+    <>
+      <Label accent='waterloo' className={styles.description__label}>
+        Assets
+      </Label>
+      <div
+        ref={ref}
+        className={cx(fieldStyles.selector, styles.selector)}
+        onClick={() => !isSearchMode && setIsSearchMode(true) && setFilter('')}
+      >
+        {!isSearchMode && (
+          <>
+            {checkedItems.length === 0 && (
+              <span className={cardStyles.ticker}>
+                Select asset for watchlist
+              </span>
+            )}
+            {checkedItems.length > 0 &&
+              checkedItems.slice(0, VIEW_ITEM_COUNT).map((item, index) => {
+                const hasDots =
+                  checkedItems.length > VIEW_ITEM_COUNT &&
+                  index === VIEW_ITEM_COUNT - 1
+                const hasComma =
+                  index < VIEW_ITEM_COUNT - 1 && index < checkedItems.length - 1
+                const seprator = hasComma ? ', ' : hasDots ? '...' : ''
+                return (
+                  <div
+                    className={cx(cardStyles.name, styles.mrhalf)}
+                    key={item.id}
+                  >
+                    {item.name}{' '}
+                    <span className={cardStyles.ticker}>{item.ticker}</span>{' '}
+                    {seprator}
+                  </div>
+                )
+              })}
+          </>
+        )}
+
+        {isSearchMode && (
+          <Input
+            autoFocus
+            maxLength='25'
+            autoComplete='off'
+            placeholder='Type to search'
+            className={styles.searchInput}
+            onChange={({ currentTarget: { value } }) => setFilter(value)}
+          />
+        )}
+
+        <Icon
+          onClick={() =>
+            isSearchMode && setFilter('') && setIsSearchMode(false)
+          }
+          type='arrow-down'
+          className={cx(
+            fieldStyles.arrow,
+            styles.arrow,
+            isSearchMode && styles.arrowup
+          )}
+        />
+
+        <Panel className={cx(styles.panel, !isSearchMode && styles.hide)}>
+          {showItems && (
+            <>
+              <h6 className={styles.groupLabel}>Contained in watchlist</h6>
+              {items.map(item => {
+                const src =
+                  isNightMode && item.darkLogoUrl
+                    ? item.darkLogoUrl
+                    : item.logoUrl
```

Comment:
- `(isNightMode && item.darkLogoUrl) || item.logoUrl`
- `[D-0-2]` Better to move it to the `AssetItem` to reduce code duplication (e.g. on `LOC 155-158`